packages/flutter_tools/lib/src/macos/cocoapods.dart

```diff
@@ -66,6 +66,13 @@ enum CocoaPodsStatus {
   brokenInstall,
 }
 
+/// Cocoapods is a depenency management solution for iOS and macOS applications.
+///
+/// Cocoapods is generally installed via homebrew and interacted with via
```

Comment:
I believe our official recommendation is to use rubygems, e.g. https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/macos/cocoapods.dart#L44

Reply:
fyi, i think the homebrew formula for cocoapods just delegates to rubygems anyway

Reply:
+1 I have never installed it with homebrew.
https://flutter.dev/docs/get-started/install/macos#deploy-to-ios-devices

Reply:
TIL, why did I install it with brew - perhaps that was old documentation?

Reply:
Done, and added link to iOS getting started docs